docs/development/pigs/pig-21.rst

```diff
@@ -0,0 +1,348 @@
+.. include:: ../../references.txt
+
+.. _pig-021:
+
+**************************************
+PIG 21 - Models Framework Improvements
+**************************************
+
+* Author: Axel Donath, Régis Terrier and Quentin Remy
+* Created: Jun 10, 2020
+* Accepted:
+* Status:
+* Discussion:
+
+Abstract
+========
+This PIG outlines further minor improvements to the modeling framework of Gammapy.
+This includes introduction of a spectra models with norm parameters, minimal
+support for energy dependent spatial models as well as simplifications of the
+YAML serialisation.
+
+
+Proposal
+========
+
+Spectral Norm Models
+--------------------
+
+For the purpose of adjusting template based models we propose to introduce a new
+class of spectral models. Those spectral models feature a norm parameter instead
+of amplitude and are named using the ``NormSpectralModel`` suffix:
+
+.. code::
+
+    from gammapy.modeling.models import (
+        PowerLawNormSpectralModel,
+        LogParabolaNormSpectralModel,
+        PiecewiseBrokenPowerlawNormSpectralModel,
+        ConstantNormSpectralModel,
+    )
+
+    pwl_norm = PowerLawNormSpectralModel(norm=, tilt=, reference=)
+    log_parabola_norm = LogParabolaNormSpectralModel(norm=, alpha=, beta=)
+    bpwl_norm = PiecewiseBrokenPowerlawNormSpectralModel(norms=, energies=)
+    const_norm = ConstantNormSpectralModel(norm=)
+
+    # typically used like
+	template = TemplateSpectralModel()
+    model = template * pwl_norm
+
+    model = BackgroundModel(
+		map=map,
+		spectral_model=pwl_norm
```

Comment:
Isn't it simpler and more intuitive that `BackgroundModel` takes a `spectral_norm` parameter rather than a `SpectralModel` which are assumed to have a flux unit.

Reply:
The `BackgroundModel` should take a `SpectralNormModel` of course...will correct this...